euphoria-operator-testkit/src/main/java/cz/seznam/euphoria/operator/test/JoinTest.java

```diff
@@ -101,6 +143,45 @@ public void batchJoinOuter() {
       public List<Pair<Integer, String>> getUnorderedOutput() {
         return Arrays.asList(
             Pair.of(0, "0+null"),
+            Pair.of(2, "2+12"), Pair.of(2, "2+12"), Pair.of(4, "4+14"),
+            Pair.of(1, "1+11"), Pair.of(1, "1+11"),
+            Pair.of(3, "3+13"), Pair.of(3, "3+13"));
+      }
+    });
+  }
+
+  @Processing(Processing.Type.BOUNDED)
```

Comment:
Why do we run these tests only for bounded datasets?

Reply:
OK, I see, the reason is probably that unbounded dataset cannot be join using global windowing. There are separate tests for windowing.